src/shipit_dashboard/shipit_dashboard/__init__.py

```diff
@@ -14,6 +14,20 @@
 HERE = os.path.dirname(os.path.abspath(__file__))
 APP_SETTINGS = os.path.abspath(os.path.join(HERE, '..', 'settings.py'))
 
+# Tasckcluster scopes
+SCOPES_USER = [
+    'project:shipit:user',
+    'project:shipit:analysis/use',
+    'project:shipit:bugzilla'
+]
+SCOPES_ADMIN = SCOPES_USER + [
+    'project:shipit:admin',
+]
+SCOPES_BOT = [
+    'project:shipit:bot',
+    'project:shipit:analysis/manage',
+]
+
```

Comment:
would it make more sense to create unique scope for each api endpoint that needs to be not public and then assign/group those scopes with roles? this would mean that you would not have to change any code if you want to change what certain role can do.